openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -367,6 +375,7 @@ def schedules_for_bin(self):
 
                     # This is used by the bulk email optout policy
                     'course_ids': [str(enrollment.course_id)],
+                    'unsubscribe_url': unsubscribe_url,
```

Comment:
Since we're already having to alter this code to send and support the unsubscribe capability, could you also include this `unsubscribe_url` in the [`edx_ace/common/base_body.html`](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html#L174-L175)?

Reply:
👍